laravel-app/app/Http/Controllers/FollowController.php

```diff
@@ -3,11 +3,30 @@
 namespace App\Http\Controllers;
 
 use App\Models\User;
+use Illuminate\Contracts\View\View;
 use Illuminate\Http\RedirectResponse;
 
 class FollowController extends Controller
 {
-    public function follow(User $user): RedirectResponse
+    /**
+     * To display all the followers-following of the user
+     */
+    public function show(User $user): View
+    {
+        $followers = $user->followers()->get();
+        $followings = $user->following()->get();
```

Comment:
Question
1.) Are you getting all of the followers of the specific user or getting all of the followers of the different users?
2.) Same question with the following?

$user->followers()->get(); ->>>> This means you are getting all of the followers of any users.
$user ->followers() ->>>>> This means you are getting all of the followers of the specific user.

Reply:
$user->followers()->get();
sir doesn't this return all the followers of the instance I passed in the variable $user? So it correctly retrieves all the followers of this specific user. Is this correct sir?

According to: 
https://stackoverflow.com/questions/68017966/im-trying-to-get-the-followers-of-a-specific-user-with-eloquent-from-a-many-to

Reply:
If it's properly working based on what you want, I will save this as resolved 🙇